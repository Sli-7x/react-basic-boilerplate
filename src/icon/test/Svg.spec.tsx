import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { SvgRuler, SvgBath, SvgPillow } from '../Svg';

describe('Svg', () => {
  it('should render correctly SvgRuler', () => {
    const tree = renderer.create(<SvgRuler />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render correctly SvgBath', () => {
    const tree = renderer.create(<SvgBath />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render correctly SvgPillow', () => {
    const tree = renderer.create(<SvgPillow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
