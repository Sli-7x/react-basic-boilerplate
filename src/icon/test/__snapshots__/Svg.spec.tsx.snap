// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svg should render correctly SvgBath 1`] = `
<svg
  height="100%"
  id="bc-svg-bath"
  viewBox="0 0 512 512"
  width="100%"
>
  <path
    d="M505 208.8l-2.4-14.3-13.3 5.8c-.4.2-10.6 4.5-29.4 9.9V80.5c0-28.1-22.9-51-51-51s-51 22.9-51 51v19.8h-28.3v28.3h79.3v-28.3h-28.3V80.5c0-15.6 12.7-28.3 28.3-28.3 15.6 0 28.3 12.7 28.3 28.3v135.7c-73.1 17.4-219.4 36.5-416.1-16.4L9 196.6l-2 12.2c-11.9 71.7 1.1 132.3 37.7 175.5 17.6 20.7 40.1 37.2 67.4 49.4-7.5 9.6-11.2 19.1-12.9 26.1H74.8v22.6h45.3v-11.3c0-.2.6-14.6 14.6-28.6C169 454 209.4 459.8 256 459.8c46.6 0 87.1-5.8 121.3-17.3 13.9 13.8 14.5 28 14.6 28.6v11.3h45.3v-22.6h-24.4c-1.8-7-5.5-16.5-13-26.1 27.3-12.2 49.8-28.7 67.4-49.4 36.7-43.1 49.7-103.8 37.8-175.5zm-55 160.9c-17.2 20.3-40.2 35.8-68.4 46.9l-.1-.1-.1.2c-34.1 13.4-75.8 20.5-125.4 20.5-49.6 0-91.3-7-125.4-20.5l-.1-.2-.1.1C102.2 405.5 79.1 390 62 369.7c-30.1-35.5-41.9-85.4-34.4-144.9 245.8 63.3 410 18 457.1 1.5 7 58.8-4.8 108.2-34.7 143.4z"
  />
</svg>
`;

exports[`Svg should render correctly SvgPillow 1`] = `
<svg
  height="100%"
  id="bc-svg-pillow"
  viewBox="0 0 512 512"
  width="100%"
>
  <path
    d="M491.6 371.4c10.8-20.9 14.7-46.7 14.7-78.3 0-16.9-1.1-35.6-2.9-56-4.5-52.2-13.4-87.4-28.7-112.2C485 110.4 490.4 96 490.4 84c0-8.6-2.8-16-8.5-21.2-5.1-4.7-12-6.9-19.9-6.5-13.4.7-29.3 9.1-43.9 23.4-39-14.2-94.4-15-174.6-11.4C167.3 71.6 115 84.4 80 108.4 62.8 95.3 44.3 88.4 29.9 90c-7.9.8-14.3 4.1-18.6 9.6-4.1 5.2-9.7 16.7-.7 35.9 4.9 10.4 13.2 21 23.6 30.2-8.1 20.5-12 44.6-12 74.8 0 12 .6 25.1 1.8 39.2 4.1 47.4 9.9 83.1 25.5 110.7-8.6 13.4-13.1 26.5-13.1 37.5 0 8.9 2.9 16.5 8.9 21.8 4.7 4.2 10.7 6.2 17.4 6.2 12 0 26.3-6.4 40.1-18.5 36.9 15.7 91 19.4 173 12.1C360 442 407.5 433 440 418.1c20.8 16.1 41.9 19.6 54.1 8.4 5.1-4.6 7.6-11.5 7.6-19.9 0-7.6-2-16.4-6.1-26.1-1.5-3.8-2.8-6.6-4-9.1zm-23.8 6.9c1.7 2.4 3.2 5.5 5.7 11.5 3.9 9.1 4.5 15.5 4.5 17.9l-.1.9c-2 1.4-13.7-.8-28.1-13.8l-6.2-5.6-7.4 3.9c-29.2 15.3-76.3 24.7-162.5 32.4-82.2 7.3-135.5 3.1-167.8-13.2l-7.8-4-6.2 6.3c-15.4 15.7-28.6 18.3-30.6 17-.4-.4-.7-1.6-.7-3.4 0-5.4 2.8-16.5 13-30.3l5-6.8-4.6-7C58 359.9 52.1 324 48.1 277.5c-1.1-13.2-1.7-25.4-1.7-36.6 0-31 4.4-54.9 13.4-73.7l4.3-8.9-7.9-5.9c-1.1-.8-2.2-1.7-3.3-2.6-9.2-7.5-16.6-16.6-20.5-24.7-2.1-4.4-2.5-7.4-2.5-9.1l.3-1.6c2.3-2.2 18.9-.6 38.3 15.3 1.1.9 2.1 1.8 3.2 2.7l7.6 6.8 8-6.3c30.3-24.1 81.8-37.4 157.5-40.7 82.3-3.6 136.8-2.7 171.8 12.6l8 3.5 5.8-6.5c17.2-19.5 33-22.9 35.4-21.3.4.4.7 1.5.7 3.1 0 5.7-3.4 18.2-15.7 33l-6 7.2 5.5 7.6C466 152.8 475 186 479.6 239.1c1.7 19.4 2.7 37.3 2.7 53.4 0 30.6-3.8 55-14.7 72.6l-4.1 6.7 4.3 6.5z"
  />
  <path
    d="M91 171c.4.8 37 82 21 200.9l23.8 3.2c17.1-126.3-21.3-210.7-23-214.2L91 171z"
  />
</svg>
`;

exports[`Svg should render correctly SvgRuler 1`] = `
<svg
  height="100%"
  id="bc-svg-ruler"
  viewBox="0 0 512 512"
  width="100%"
>
  <path
    d="M0 26.6v458.7h512L0 26.6zM23 78l429 384.4h-96.2v-51.6h-23v51.6h-68.9v-51.6h-23v51.6h-57.4v-51.6h-23v51.6h-57.4v-51.6h-23v51.6H23V78z"
  />
  <path
    d="M80.3 222.5v143.3h180.4L80.3 222.5zm23 47.5l91.7 72.8h-91.7V270z"
  />
</svg>
`;
